.github/workflows/python-package.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Sorry for commenting multiple times.
I think we need to wrap these values by quotes. It looks the CI unexpectedly interpreted `3.10` as `3.1`.
```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```